conans/client/command.py

```diff
@@ -854,6 +856,12 @@ def remove(self, *args):
             self._cache.remove_locks()
             self._user_io.out.info("Cache locks removed")
             return
+        elif args.system_reqs:
+            if not ref:
+                raise ConanException("Please specify a valid package reference to be cleaned")
+            self._cache.remove_package_system_reqs(ref)
+            self._user_io.out.info("Cache system_reqs from %s has been removed" % str(ref))
```

Comment:
that appears to be misleading, you're ignoring errors from shutil, but always output that system requirements have been removed. what if they have failed to remove? why not output error message in that case?

Reply:
I've followed the same behavior from locks, but you are right. If some error occur, like permission, this command will not make any effect.

Reply:
Done!